draft-ietf-spice-sd-cwt.md

```diff
@@ -158,7 +158,233 @@ Full Disclosure
+    <<[
+        <<[
+            /salt/   h'8d5c15fa86265d8ff77a0e92720ca837',
+            /claim/  500,  / age_at_least_18 /
+            /value/  true
+        ]>>,
+        <<[
+            /salt/   h'd84c364fad31e0075213141ca7d1408f',
+            /claim/  501,  / age_at_least_21 /
+            /value/  false
+        ]>>,
+        <<[
+            /salt/   h'30eef86edeaa197df7bd3d17dd89cd87',
+            /claim/  "region",
+            /value/  "ca" /California/
+        ]>>,
+        <<[
+            /salt/   h'284538c4a1881fac49b2edc550c1913e',
+            /claim/  "post_code",
+            /value/  "94188"
+        ]>>,
+        <<[
+            /salt/   h'86c84b9c3614ba27073c7e5a475a2a13',
+            /value/  "4.1.7"
+        ]>>
+    ]>>
+  },
+  / payload / << {
+    / iss / 1  : "https://issuer.example",
+    / sub / 2  : "WRQ2RbY5RYJCIxfDQL9agl9fFSCYVu4Xocqb6zerc1M",
+    / exp / 4  : 1725330600, /2024-09-02T19:30:00Z/
+    / nbf / 5  : 1725243840, /2024-09-01T19:25:00Z/
+    / iat / 6  : 1725244200, /2024-09-01T19:30:00Z/
+    / cnf / 8  : {
+      / cose key / 1 : {
+        / alg: ES256 /  3: -7,
+        / kty: EC2   /  1: 2,
+        / crv: P-256 / -1: 1,
+        / x / -2: h'hVTrJ13Nb70cesZBqiyQ2SAi_Q0wJLWvGMfMYa1Sei0',
+        / y / -3: h'TceuLGd-ltDMgll2Vc6S1VA_VCk9h4ddHnnOR3AZQ0M'
+      }
+    },
+#    / sd_hash / 11       : h'abcdef12', / TODO: fix ??? /
+    / sd_alg /  12       : -16, / SHA-256 /
+    / redacted_keys / 13 : [
+        / redacted age_at_least_18 /
+        h'7e6e350907d0ba3aa7ae114f8da5b360' +
+        h'601c0bb7995cd40049b98e4f58fb6ec0',
+        / redacted age_at_least_21 /
+        h'1e7275bcda9bc183079cd4515c5c0282' +
+        h'a2a0e9105b660933e2e68f9a3f40974b'
+    ],
+    / swversion / 271 : [
+      "3.5.5",
+      /redacted version "4.1.7" /
+      { "...":  h'a0f74264a8c97655c958aff3687f1390' +
+                h'ed0ab6f64cd78ba43c3fefee0de7b835' }
+    ],
+    "address": {
+        "country" : "us",            / United States /
+        /redacted_keys/ 13 : [
+            / redacted region /
+            h'c47e3b047c1cd6d9d1e1e01514bc2ec9' +
+            h'ed010ac9ae1c93403ec72572bb1e00e7',
+            / redacted post_code /
+            h'0b616e522a05d8d134a834979710120d' +
+            h'41ac1522b056d5f9509cf7e850047302'
+        ]
+    }
+  } >>,
+  / signature / h'3337af2e66959614' /TODO: fix /
+])
+~~~
+
+Some of the claims are *redacted* in the payload. The corresponding
+*disclosure* is communicated in the unprotected header in the `sd_claims`
+key. For example, the `age_at_least_18` claim is a Salted Disclosed Claim,
+consisting of a per-disclosure random salt, the claim name, and claim value.
+
+~~~ cbor-diag
+<<[
+    /salt/   h'8d5c15fa86265d8ff77a0e92720ca837',
+    /claim/  500,  / age_at_least_18 /
+    /value/  true
+]>>,
+~~~
+
+This is represented in hex by the CBOR byte string value:
+
+~~~
+5683508D5C15FA86265D8FF77A0E92720CA8371901F4F5
+~~~
```

Comment:
```suggestion
```

I don't think that including a hex CBOR representation enhances the readers' comprehension of the text.  The CDDL above does the job fine by itself.  I'd delete the hex CBOR.